org.tenacityaudio.Tenacity.yaml

```
@@ -0,0 +1,209 @@
+app-id: org.tenacityaudio.Tenacity.yaml
+default-branch: stable
+# freedesktop-sdk should work too, but causes problems with
+# the timeline mouse cursor on some systems for unkown reasons
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: tenacity
+rename-desktop-file: tenacity.desktop
+rename-icon: tenacity
+#rename-appdata-file: tenacity.appdata.xml
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network # Audacity uses network sockets for IPC
+  - --socket=pulseaudio
+  - --device=all # ALSA
+  - --filesystem=host
+  - --env=LD_LIBRARY_PATH=/app/lib/tenacity
+  - --env=ALSA_CONFIG_PATH=
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa
+  - --env=LV2_PATH=/app/extensions/Plugins/lv2
+  # Audacity uses VST_PATH
+  - --env=VST_PATH=/app/extensions/Plugins/lxvst
+  # JACK support via pipewire
+  - --filesystem=xdg-run/pipewire-0
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+
+add-extensions:
+  org.audacityteam.Audacity.Codecs:
+    directory: lib/codecs
+    add-ld-path: lib
+    bundle: true
+    autodelete: true
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: '20.08'
+    add-ld-path: lib
+    merge-dirs: ladspa;lv2;lxvst
+    subdirectories: true
+    no-autodownload: true
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/aclocal
+  - /share/man
+  - '*.la'
+  - '*.a'
+modules:
+
+  - name: wxwidgets
+    rm-configure: true
+    config-opts:
+      - --with-libpng
+      - --with-zlib
+      - --with-cxx=14
+      - --disable-sdltest
+      - --disable-webkit
+      - --disable-webview
+      - --disable-webviewwebkit
+      - --disable-ribbon
+      - --disable-propgrid
+      - --disable-richtext
+      - --with-expat=builtin
+      - --with-libiconv=/usr
+    cleanup:
+      - /bin
+      - /lib/wx
+      - /share/bakefile
+    sources:
+      - type: git # they want their fork
+        url: https://github.com/audacity/wxWidgets
+        # branch: audacity-fixes-3.1.3
+        tag: v3.1.3.1-audacity
```

```suggestion
        url: https://github.com/wxWidgets/wxWidgets
        # branch: audacity-fixes-3.1.3
        tag: v3.1.5
```
Required for high DPI support